executable-monitor/executable-monitor.py

```diff
@@ -36,8 +149,8 @@
     parser.add_argument('--success-line',
                         type=str,
                         required=False,
-                        help='Line that indicates executable completed successfully. Required if --success-exit-status is not used.')
-    parser.add_argument('--success-exit-status',
+                        help='Line that indicates executable completed successfully. Required if --success-exit-code is not used.')
+    parser.add_argument('--success-exit-code',
                         type=int,
                         required=False,
                         help='Exit status that indicates that the executable completed successfully. Required if --success-line is not used.')
```

Comment:
Don't we need these parsers as we are passing in these parameters?

Reply:
Saw that this was added later in the file. Marking this comment as resolved.

Reply:
I changed it from "success-exit-status "to "success-exit-code" is all

Reply:
I see my folly - meant to put this comment elsewhere :D